### ISSUE TITLE
Change default AdHoc server again

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#12340]
This time to myneighborsushicat.com which seems to be a working one. 

At some point we might have to maintain one... Anyone knows what goes into running one of these?

**Comment by adenovan** [comment]
https://github.com/MrColdbird/aemu/tree/master/pspnet_adhocctl_server all code here with the player status. 

that is semi working adhoc server unless player set dmz on router or hosting the server inside vpn, maintain one with this requirement is useless the adhoc experience is slow or too complicated to setup. 

thats why many player did not play over internet especially if they have limited options to open their internet port as every p2p packet is blocked due behind NAT and such. 

on my amultios branch there is exist a relay to arbitrate p2p packet over internet tunnel them over tcp. (server written on nodejs) not every game working with tunnel and its require low latency to the server the experience is not so bad for user at least user not have to setup their complicated networking hardware just to play over internet.

**Comment by unknownbrackets** [comment]
Well, to do this properly would probably require edge nodes that talk to each other.  Otherwise the server must be in some region, and people in that region are going to be the only ones with good latency...

-[Unknown]

**Comment by adenovan** [comment]
yes exactly thats why right now im working in adhoc module and integrate the feature with mqtt protocol. mqtt is easy to setup in server side as the server only act as broker for the connection the client can be independent too not rely to much on server except to arbitrate. 

most psp game expect latency to be lower than 30ms to function normally. and its time to rebase the adhoc module with many adapter like jpcsp do in their code. as adhoc feature can be tunneled over any protocol user want as long as the packet arrived on destination successfully.

**Comment by hrydgard** [comment]
Changed to myneighborsushicat for now, bumping hosting our own or similar to later.

**Comment by MojoJojoDojo** [comment]
Well http://myneighborsushicat.com/ is now dead as well ( for a full day tho).
A site maintained by someone the PPSSPP team knows would be nice.

**Comment by MojoJojoDojo** [comment]
And now its dead for over a week...
Sigh..

**Comment by hrydgard** [comment]
the sushi cat seems to be back, but the situation is indeed not ideal...

**Comment by Linblow** [comment]
socom.cc will always be there hehe

**Comment by MojoJojoDojo** [comment]
> the sushi cat seems to be back, but the situation is indeed not ideal...

At least for me and others it's dead and wont connect and it freezes the whole ppsspp when you try to connect.

**Comment by hrydgard** [comment]
who actually runs sushicat? :)

**Comment by MojoJojoDojo** [comment]
I know he goes by SushiCat on PPSSPP discord server...
That's about it.
Tried to contact him but he stopped answering recently...
